src/index.html

```diff
@@ -3,12 +3,51 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" 
+      href="https://fonts.gstatic.com" 
+      crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
```

Comment:
Generally you should add empty lines between siblings:
```html
<body>
  <div>
    <span>1</span>
  
    <p>2</p>
  </div>

  <div>
    <div>
      <button></button>  
    </div>

    <span></span>
  </div>
</body>
```

But there are exceptions for example when you have a bunch of repeated tags in container like here:
```html
<ul>
  <li><a></a></li>
  <li><a></a></li>
  <li><a></a></li>
  <li><a></a></li>
</ul>

<ul>
  <li></li>
  <li></li>
  <li></li>
  <li></li>
</ul>
```